### ISSUE TITLE
Sample application demos or guides

### ISSUE THREAD

**Issue by theNaut1lus** [frappe/gantt#139]
Hey, love the library to bits, was looking for an open source library to build gantt charts and here it is 👍 
Would love it if there was some inclusion towards adding real-life examples and demos to maybe increase the user reachability towards this library.
Many won't be searching for libraries but will be searching for step by step solutions, if we make our own solutions, more users will reach this repo and download it more.
I am also interested in starting to work on this as well 🥇

**Comment by Alex4Codes** [comment]
+++

**Comment by larspai** [comment]
Agree. A very nice repo, however the installation instructions on the readme and demo pages are not consistent (one use a div container, the other a svg), and neither of them work (npm install does not set up a structure with the relevant js- and css files, so including it in your index.html requires a bit more than just that - and what is actually required is unclear). Trying to take the demo index.html file apart to understand how to get it to work is seriously convoluted and really not the way you should be approaching good code like this...

**Comment by safwansamsudeen** [comment]
We're working on build better docs and demos - expect by the end of this year tops!

Keeping this open so that I don't forget.

**Comment by safwansamsudeen** [comment]
> expect by the end of this year tops!
:P I was slightly off, but the demo is up and will be live very soon. The docs have _just_ been pushed!
